src/PaymentGateways/Gateways/TestGateway/TestGatewayOffsite.php

```diff
@@ -32,6 +29,15 @@ class TestGatewayOffsite extends PaymentGateway
         'securelyReturnFromOffsiteRedirect'
     ];
 
+    /**
+     * @unreleased
+     */
+    public function enqueueScript(int $formId)
+    {
+        // temporary action to enqueue gateway scripts in feature plugin
+        do_action('givewp_donation_form_enqueue_test_gateway_offsite_scripts');
```

Comment:
Same thing here. Also, we're ultimately merging everything into a single test gateway, right?

Reply:
yeah possibly f4f97558